asyncprofiler/AsyncProfiler.proto

```diff
@@ -44,11 +44,17 @@ message AsyncProfilerData {
   }
 }
 
+message AsyncProfilerCollectMessage {
+    AsyncProfilerCollectType type = 1;
+}
+
 enum AsyncProfilerCollectType {
   // PROFILING_SUCCESS means the Java Agent has finished the execution
   PROFILING_SUCCESS = 0;
   // EXECUTION_TASK_ERROR means potential execution error caused by the Java Agent, such as an error in the sent task parameters.
   EXECUTION_TASK_ERROR = 1;
+  // TERMINATED_BY_OVERSIZE means that the server cannot receive the jfr file because it is too large.
```

Comment:
```suggestion
  // TERMINATED_BY_OVERSIZE means that the server cannot receive the JFR file because the size of the file is over the limitation.
```